### PR TITLE
feat(PE-3970): update searchbar design

### PR DESCRIPTION
**Pull request by atticusofsparta**
image

### CODE REVIEW

File: src/components/inputs/Search/SearchBar/SearchBar.tsx [hunk 1]
```diff
@@ -123,12 +117,15 @@ function SearchBar(props: SearchBarProps) {
...
         className="searchbar"
         style={
-          isSearchValid
-            ? !searchSubmitted || showDefaultText
-              ? { borderColor: '' }
+          !searchBarText
+            ? { borderColor: '', marginBottom: 30 }
+            : isSearchValid
+            ? !searchSubmitted
+              ? { borderColor: 'white', marginBottom: 30 }
```
Comment (karlprieb) [inline src/components/inputs/Search/SearchBar/SearchBar.tsx:128]: really hard to read that. Is not easy to have a `const` that will evaluate the right styles and use it here?

Comment (atticusofsparta) [reply src/components/inputs/Search/SearchBar/SearchBar.tsx:128]: im confused, do you mean do the same thing but move it to a const?

Comment (dtfiedler) [reply src/components/inputs/Search/SearchBar/SearchBar.tsx:128]: yeah - i think he's suggesting for such a complicated style rule, move it to a function or constant and avoid the nested ternary operators (i agree)

Comment (atticusofsparta) [reply src/components/inputs/Search/SearchBar/SearchBar.tsx]: fixed in https://github.com/ar-io/pdns-react/pull/227/commits/ef1c3d28bdea0b95c3fceabe861cf6142082e488

**Comment by dtfiedler** [comment]
image

File: src/components/inputs/Search/SearchBar/SearchBar.tsx [hunk 2]
```diff
@@ -123,86 +132,114 @@ function SearchBar(props: SearchBarProps) {
...
+            }}
+            onClick={() => {
+              isSearchValid && _onSubmit();
+            }}
+          >
+            <SearchIcon fill="white" width={18} height={18} />
+          </button>
         )}
       </div>
+      {searchSubmitted && isAvailable ? (
```
Comment (dtfiedler) [inline src/components/inputs/Search/SearchBar/SearchBar.tsx:226]: see screenshot about when a wallet is not connected

File: src/components/inputs/Search/SearchBar/SearchBar.tsx (hunk 1 above)
Comment (karlprieb) [reply src/components/inputs/Search/SearchBar/SearchBar.tsx]: Great, thanks!

File: src/components/inputs/Search/SearchBar/SearchBar.tsx (hunk 2 above)
Comment (atticusofsparta) [reply src/components/inputs/Search/SearchBar/SearchBar.tsx:226]: removed in https://github.com/ar-io/pdns-react/pull/227/commits/b6d03592496f2821c7afbb3882d90237c9944b54

File: src/components/inputs/Search/SearchBar/SearchBar.tsx [hunk 3]
```diff
@@ -123,86 +132,114 @@ function SearchBar(props: SearchBarProps) {
...
+        <button
+          className="accent-button center"
+          onClick={_onSubmitButton}
+          style={{
+            width: 130,
+            height: 50,
+            fontSize: 14,
+            padding: 0,
+            marginTop: 30,
```
Comment (dtfiedler) [inline src/components/inputs/Search/SearchBar/SearchBar.tsx:235]: why can't we use an existing button class for this? also - noticeably moves down the feature domain on the page when this shows. not a stopper - but seems glitchy

Comment (atticusofsparta) [reply src/components/inputs/Search/SearchBar/SearchBar.tsx]: the moving to featured domains is actually part of the figma - that state of the searchbar pushed it down while maintaining the gap.

Updated the class to be more generic with common styles.

changes in https://github.com/ar-io/pdns-react/pull/227/commits/86a8620bc0107969766621a5438b2912e2f1cf82

**Review by dtfiedler — APPROVED** [review]

**Pull request merged**